shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -27,7 +27,7 @@
 
 static const char* kApplicationKernelSnapshotFileName = "kernel_blob.bin";
 
-static flutter::Settings DefaultSettingsForProcess(NSBundle* bundle = nil) {
+flutter::Settings FLTDefaultSettingsForProcess(NSBundle* bundle) {
```

Comment:
Nit: this name is goofy, a bundle isn't a process.  If we are going to make this function visible we should maybe give it an appropriate name.

Reply:
Good call, will do